ts/index.ts

```diff
@@ -446,17 +453,21 @@ class Gossipsub extends Pubsub {
    * @returns {Promise<void>}
    */
   async _processRpcMessage (msg: InMessage): Promise<void> {
-    const msgID = await this.getMsgId(msg)
-    const msgIdStr = messageIdToString(msgID)
+    const fastMsgIdStr = messageIdToString(SHA256.digest(msg.data))
```

Comment:
I think the fast message id function should be pulled into a method so it may be overridden.